alkira/resource_alkira_connector_azure_vnet.go

```diff
@@ -247,6 +247,11 @@ func resourceAlkiraConnectorAzureVnet() *schema.Resource {
 				Type:        schema.TypeInt,
 				Optional:    true,
 			},
+			"direct_inter_vnet_communication_group": {
+				Description: "Direct inter-vnet communication group.",
```

Comment:
I think we need to explain this a bit, since it's referring to the resource `alkira_group_direct_inter_connector`. The group name is generic. It's not specific to just inter VNET. Do we plan to have like inter VPC-VNET?

Reply:
I don't think direct VPC-VNET would be possible since they are on different cloud and all this feature does is allow two VPCs or VNET in the same cloud to use the cloud backbone.